### PR TITLE
Add pt-br/docs/reference/glossary/cluster-infrastructure.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translate pt-br/docs/reference/glossary/cluster-infrastructure.md

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/cluster-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: infraestrutura de Cluster
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/cluster-infrastructure.md:2]: I think "infraestrutura" here should have the first letter capitalized.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Minor capitalization.

**Comment by edsoncelio** [comment]
Thanks!
/approve

**Pull request merged**